proto/evmos/incentives/v1/incentives.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+package evmos.incentives.v1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/bank/v1beta1/bank.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/tharsis/evmos/x/incentives/types";
+
+// Incentive defines an instance that organizes distribution conditions for a given
+// smart contract
+message Incentive {
```

Comment:
Distribution of users with incentives:

`key -> value` for store:
```
prefix + `incentivized_contract_address_bytes` + `user_address_bytes` -> cummulative gas spent during the epoch
```


```protobuf
message Rewards {
// hex address of the incentivized contract
string contract_address = 1;
// user address that interacts with the incentive
string user_address = 2;
// cumulative gas spent during the epoch
uint64 cummulative_gas = 3;
}
```

Reply:
Added the rewards message, will look at storing the gas when we implrement the hook